Mapbox/Configurations/Mapbox.xcconfig

```diff
@@ -1,11 +0,0 @@
-// Default configuration used for the base projects.
-#include "base.xcconfig"
-
-BUILD_LIBRARY_FOR_DISTRIBUTION = YES
```

Comment:
We don't want this in the Xcode project anymore since it now uses SPM. For packager, this flag is passed to xcodebuild on the command line. Nothing else used this config file.